frontend/awx/resources/templates/WorkflowVisualizer/hooks/useWorkflowVisualizerToolbarActions.tsx

```diff
@@ -24,22 +25,28 @@ import {
 import { WorkflowNode } from '../../../../interfaces/WorkflowNode';
 import { useBulkConfirmation, usePageNavigate } from '../../../../../../framework';
 import { AwxRoute } from '../../../../AwxRoutes';
-import { getItemKey } from '../../../../../common/crud/Data';
+import { getItemKey, postRequest } from '../../../../../common/crud/Data';
 import { stringIsUUID } from '../../../../common/util/strings';
 import { AddNodeButton } from '../components/AddNodeButton';
 import getDocsBaseUrl from '../../../../common/util/getDocsBaseUrl';
 import { useAwxConfig } from '../../../../common/useAwxConfig';
 
 export function useWorkflowVisualizerToolbarActions(nodes: WorkflowNode[]) {
   const { t } = useTranslation();
+  const params = useParams<{ id: string }>();
   const [isExpanded, setIsExpanded] = useState<boolean>(false);
   const [isKebabOpen, setIsKebabOpen] = useState<boolean>(false);
   const pageNavigate = usePageNavigate();
   const config = useAwxConfig();
-  const params = useParams<{ id: string }>();
   const bulkAction = useBulkConfirmation<WorkflowNode>();
+  const handleLaunchWorkflow = useCallback(async () => {
+    await postRequest(`/api/v2/workflow_job_templates/${Number(params.id).toString()}/launch/`, {});
+    setIsKebabOpen(false);
```

Comment:
If we pass `onOpenChange={(isOpen: boolean) => setIsKebabOpen(isOpen)}` to the Dropdown component,  then we don't have to call `setIsKebabOpen(false)` in handleLaunchWorkflow or handleRemoveAllNodes.